src/env.cpp

```diff
@@ -1407,6 +1407,13 @@ maybe_t<env_var_t> env_stack_t::get(const wcstring &key, env_mode_flags_t mode)
             wcstring_list_t result;
             if (history) history->get_history(result);
             return env_var_t(L"history", result);
+        } else if (key == L"pipestatus") {
+            auto&& js = proc_get_last_job_statuses();
```

Comment:
const?

Reply:
That is a forwarding reference, having it const does not make sense.
But I agree about const, so will change it to `const auto&'